diffrax/_term.py

```diff
@@ -776,3 +785,136 @@ def _to_vjp(_y, _diff_args, _diff_term):
         dy, vjp = jax.vjp(_to_vjp, y, diff_args, diff_term)
         da_y, da_diff_args, da_diff_term = vjp((-(a_y**ω)).ω)
         return dy, da_y, da_diff_args, da_diff_term
+
+
+# The Underdamped Langevin SDE trajectory consists of two components: the position
+# `x` and the velocity `v`. Both of these have the same shape. So, by LangevinX we
+# denote the shape of the x component, and by LangevinTuple we denote the shape of
+# the tuple (x, v).
+_LangevinX = PyTree[Shaped[Array, "?*langevin"], "LangevinX"]
+_LangevinTuple = tuple[_LangevinX, _LangevinX]
+_LangevinBM = TypeVar("_LangevinBM", bound=Union[_LangevinX, AbstractBrownianIncrement])
```

Comment:
I don't understand this type variable -- it seems to go unused below. (I.e. I don't see `_LangevinDiffusionTerm[...]` anywhere.)

I think you probably just want to inline this `Union` into `_LangevinDiffusionTerm(AbstractTerm[...]):`?

Reply:
Good point! I think this is a remnant of when I was first coding this and Python's type system felt pretty foreign to me (it still does sometimes).